package.json

```diff
@@ -25,7 +25,7 @@
     "lambdas": "cd packages/lambdas && npm run $SCRIPT && cd ../..",
     "commit": "cz",
     "check-secrets": "docker run --rm -v $(pwd):/path zricethezav/gitleaks:v8.17.0 protect --source='/path' --staged --no-banner -v",
-    "publish:alpha": "npm run build && npx lerna publish --dist-tag next prerelease",
+    "publish:alpha": "npm run build && npx lerna publish --dist-tag next",
```

Comment:
OOC - why is `prerelease` being removed? I thought it was necessary

Reply:
I should've mentioned this on the PR, sorry.

`prerelease` is a specific version definition, so its like telling Lerna to bump the alpha number w/o touching major, minor, or patch. This is an assumption that's not valid, as the developer might want to set the next version to `preminor`, for example (it bumps minor but b/c is a "pre" version it will have `-alpha...` at the end.

https://github.com/lerna/lerna/tree/main/libs/commands/version#prerelease